### ISSUE TITLE
95s visuals draft 1

### ISSUE THREAD

**Issue by jessmhoehner** [USAID-OHA-SI/thathill#1]
- is there clustering around countries 
who have met which 95?
- scatterplot, quadrants
- scatterplot actual ests too
- error bands 
- strip plot, distance

**Comment by tessam30** [comment]
- [ ] Cluster analysis steps
  - [ ]   Calculate a binary variable flagging OUs that have a complete cascade for 2021
  - [ ]   Create a subset dataset that includes the OU name, the estimates, the indicator, and the lower bound
  - [ ]  Until we decide upon a better way, we can use the lower bound of the estimate as an additional piece of information to pass to the clustering algorithm. We want to capture how much uncertainty is associated with the estimates and this additional piece of data can allow for a crude attempt at this (open to other suggestions).
  - [ ] Run the clustering algorithm on the 6 columns (Known status, Know status on ART, On ART VLS, repeat that list with errors)
  - [ ] plot the results and determine optimal # of clusters -- from earlier look at things, 5 seems about right
  - [ ] Join the estimated cluster values back into the dataset and plot the results. Using `plot_ly()`, you can create an interactive 3-d view (for exploratory views only). 
- [ ]  TODO: determine how we describe each cluster (some are intuitive (South Africa / Brazil), others will need a little brainstorming.

**Comment by tessam30** [comment]
Sample output 
![UNAIDS_cascade_cluster](https://user-images.githubusercontent.com/5873344/183300253-2c827fde-43a5-429b-8d6e-489cbadae3f6.png)

**Comment by jessmhoehner** [comment]
I have added an example PCA with some preliminary interpretation in the comments in explore_unaids.R on the main branch. The countries which are clustered here also appear to cluster in the k-means analysis but I find the PCA much more interpretable, assuming my interpretation makes sense to you. What I am seeing in the PCA aligns with what I think we saw in exploratory analysis of the data and what seems to make sense clinically but would leave up to your best judgement. 

![image](https://user-images.githubusercontent.com/44708009/184906248-ffda229a-41ab-48cd-a66f-00480f076eaf.png)
